src/Controller/WhoWeWorkWithController.php

```diff
@@ -198,9 +198,15 @@ public function whoWeWorkWithAction(Request $request) : Response
             [
                 'name' => 'PubMed Central',
                 'filename' => 'pmc',
-                'type' => 'image/png',
+                'type' => 'image/svg+xml',
```

Comment:
I can't see the `jisc.svg` in here and can we not delete the `pmc.png` file?

Reply:
I can see `jisc.svg` on line 175, so seems it's just adding the missing image file. Agree that the `pmc.png` file can be deleted now (if it hasn't already)

Reply:
I have removed the png and the jisc.svg should be in there as I have committed it.